SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/rest/component/RestModule.java

```diff
@@ -37,4 +46,19 @@ SeriesLoader provideSeriesLoader(SeriesLoaderImpl impl) {
     RequestFactory provideRequestFactory(RequestFactoryImpl impl) {
         return impl;
     }
+
+    @Provides
+    SeriesRequest provideSeriesRequest(EPGuideApi epGuideApi, OmdbApi omdbApi) {
```

Comment:
We use Dagger's Provider<T> in request factory which makes these provide methods unnecessary as we have provide methods for all dependencies.